### PR TITLE
DP-4211 - Video - making updates backward compatible.

### PR DESCRIPTION
**Pull request by gender404**
Added a .ma__video--new class to the video pattern so old styles can be applied if it has not been added.  Eventually we can remove this class with the next Major release.

### CODE REVIEW

**Review by jesconstantine — APPROVED** [review]
@legostud If I am understanding it correctly, this PR is creating an opt-in (via class) accessibility presentation fix to the video atom pattern and without the opt-in the video pattern will render as it does in production today.  Thanks for the work around!

File: styleguide/source/assets/scss/01-atoms/_video.scss [hunk 1]
```diff
@@ -15,11 +15,55 @@
     }
   }
 
+  // Backward Compatible - had to add the --new as a variant for the new styling
   .main-content--full .page-content > & {
-    @include ma-container();
+    @include ma-container;
   }
 
   &__link {
```
Comment (jesconstantine) [inline styleguide/source/assets/scss/01-atoms/_video.scss:23]: See [comment in release PR](https://github.com/massgov/mayflower/pull/573/files#r136220998) about making this backwards compatible.

**Comment by gender404** [comment]
It's an ugly hack, but it was the only way to make it work.  We can remove the dependency for that class in the next Major release.

File: styleguide/source/assets/scss/01-atoms/_video.scss (hunk 1 above)
Comment (gender404) [reply styleguide/source/assets/scss/01-atoms/_video.scss:23]: good catch

**Pull request merged**